packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testOpenPreferences.st

```diff
@@ -0,0 +1,16 @@
+as yet unclassified
+testOpenPreferences
+
+	| prefBrowser prefBrowserMorph |
+
+	klass new openPreferences.
+
+	prefBrowserMorph := [World submorphs detect: [ :morph | morph class name = #PreferenceBrowserMorph]]
+		on: NotFound
+		do: [:exc | Transcript show: exc messageText; cr. ^nil].
```

Comment:
Exactly! After calling 'openPreferences' an instance of PreferenceBrowserMorph should pop up. 'World submorphs' contain this instance. If not 'NotFount' is thrown and nil is returned.